analytics/js_tracking_api.rst

```diff
@@ -839,3 +839,19 @@ Example of usage::
 
     _paq.push(["trackSiteSearch", "test", false, 20]);
 
+
+
+Tracking user anonymously
+``````````````````````````
+To track visitor anonymously (without consent) use the ``setUserIsAnonymous`` function::
+
+    _paq.push(["setUserIsAnonymous", isAnonymous]);
+
+.. describe:: isAnonymous
+
+    **Required** ``integer/boolean`` Flag that sets anonymous tracking on and off (true/false or 1/0)
+
+Example of usage::
+
+    _paq.push(["trackSiteSearch", true]);
+
```

Comment:
I see that RST format has a [versionadded](http://www.sphinx-doc.org/en/stable/markup/para.html#directive-versionadded) directive to notify user when element was introduced. Maybe we should use that at the end of description? We didn't use it yet, but you can find `deprecated` directive in this file that conveys similar information.